draft/overlay/oledbonnet.md

```diff
@@ -0,0 +1,80 @@
+﻿
+<!--
+---
+name: OLED Bonnet
+class: board
+type: Display
+formfactor: pHAT
+manufacturer: Adafruit
+description: A 128x64 display with jostick and buttons for your Pi
+url: https://learn.adafruit.com/adafruit-128x64-oled-bonnet-for-raspberry-pi
+github: https://github.com/adafruit/Adafruit_Python_SSD1306
+buy: https://www.adafruit.com/product/3531
+image: 'oledbonnet.png'
+pincount: 40
+eeprom: no
+power:
+  '1': 
+ground:
+  '6':
+  '9':
+  '14':
+  '20':
+  '25':
+  '30':
+  '34':
+  '39':
+
+pin:
+  '3':
+    mode: i2c
+  '5':
+    mode: i2c
+  
+  ’27’: 
```

Comment:
I think you have a bunch of 'smart quotes' in there, I'll fix but try to process and test your overlays before submit if possible, these ones almost certainly will crash our not-so-smart rendering script.